input_generation/generateBatchInput.ts

```diff
@@ -20,13 +20,23 @@ if (path.extname(targetFile) != '.json') {
 }
 
 const configFile = JSON.parse(fs.readFileSync(hyparConfigPath))
-const parameters = configFile.inputs
+const parameters = configFile.input_schema.properties
```

Comment:
Is it critical for us to remove support for inputs? Even if it's just a complete copy of the old code in an `if(configFile.inputs)` it would be ok.

Reply:
Done.